File: Apollo-Library/src/main/java/me/solar/apolloLibrary/Valid.java
package me.solar.apolloLibrary;

public class Valid {

    private Valid() {
        throw new IllegalStateException("Utility class");
    }

    public static boolean isNull(Object object) {
        return object == null;
    }

    public static boolean isNotNull(Object object) {
        return !isNull(object);
    }

    public static void checkBoolean(boolean bool, String message) {
        if (!bool) {
            throw new IllegalArgumentException(message);
        }
    }

}

File: ApolloCore/src/main/kotlin/me/solar/apollo/apolloBukkitCore/utils/Valid.kt
package me.solar.apollo.apolloBukkitCore.utils

object Valid {
    fun checkBoolean(value: Boolean, message: String) {
        if (!value) {
            throw IllegalArgumentException(message)
        }
    }

    fun checkNotNull(value: Any?, message: String) {
        if (value == null) {
            throw IllegalArgumentException(message)
        }
    }

    fun checkNotEmpty(value: String) {
        checkNotNull(value, "Value cannot be null")
    }
}